corehq/apps/reports/management/commands/project_stats_report.py

```diff
@@ -82,18 +136,23 @@ def handle(self, domain, months, csv, **options):
             .domain(domain).values_list("_id", flat=True)
         )
 
+        self._doc_counts()
```

Comment:
These methods are setting stats, but they are simply called with a noun, so it might not be easy for someone to understand what the method is doing without reading the method. I would rename them to `set_<stat_name>` or something on those lines. Also `_` at the beginning might not be necessary in this case.

Just a stylistic comment, nothing major.